libnymea/interfaces/mediaplayer.json

```diff
@@ -1,11 +1,11 @@
 {
+    "description": "Media player interface. Used by devices/services which can play back media.",
     "extends": "media",
     "states": [
         {
             "name": "playbackStatus",
             "type": "string",
-            "allowedValues": ["Playing", "Paused", "Stopped"],
-            "writable": true
+            "allowedValues": ["Playing", "Paused", "Stopped"]
```

Comment:
states have `possibleValues` instead of `allowedValues`

Reply:
this is currently implemented as "allowedValues" in the core logic... would need to change it in all interfaces, so, cannot easily change it here only

Reply:
The device class does not get loaded with allowedValues, so it breaks the plugin!